README.md

```diff
@@ -1,7 +1,108 @@
 # timechop
-turn feature tables into design matrices for temporal cross validation
+Generate temporal cross-validation time windows for matrix creation
```

Comment:
I've been more and more just using "temporal validation." @rayidghani  has a strong objection to "cross validation" in this context, unless that's changed.